tox.ini

```diff
@@ -16,6 +16,7 @@ description =
     flake8: Run static analysis
     docs: Generate documentation
     pkg: Verify the package
+    system_tests: Run system tests (requires driver to be installed)
```

Comment:
this refers to the driver runtime exclusively, right? (not the bindings). Can you clarify?

Reply:
Added runtime